pkg/odo/cli/url/describe.go

```diff
@@ -55,29 +57,45 @@ func (o *URLDescribeOptions) Validate() (err error) {
 
 // Run contains the logic for the odo url list command
 func (o *URLDescribeOptions) Run() (err error) {
+	if experimental.IsExperimentalModeEnabled() {
+		u, err := url.GetIngress(o.KClient, o.EnvSpecificInfo, o.url)
```

Comment:
don't we need state sync like below here?

Reply:
This PR is for the basic usage of odo orl devfile support. I've noticed the missing of the `state`. I've create another issue to track that: https://github.com/openshift/odo/issues/2787

Reply:
OK :+1: